src/main/resources/stubs/github.com/scionproto/scion/go/lib/assert/assert.gobra

```diff
@@ -1,13 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
why do you want to remove this file? There is a test case that relies on it

Reply:
Mostly, because it does not make sense to have this file as a stub, given that we expect to never use it

Reply:
Because we prove that an assertion always holds and thus we don't need to check it at runtime?

Reply:
What do you mean?
Actually, this file does not seem to be used in any project (not even scion), and it is also not a built-in or part of the std-lib. As such, having builtin support in Gobra to `import  "github.com/scionproto/scion/go/lib/assert"` is unnecessary.

However, I will keep this file but move to the tests such that we can keep the tests that depend on it

Reply:
I think I (?) / we have included it because SCION was using it. If it no longer thus then we can definitely remove it.